eclipse-scout-chart/src/chart/ChartJsRenderer.js

```diff
@@ -1495,14 +1497,37 @@ export default class ChartJsRenderer extends AbstractChartRenderer {
     }
   }
 
-  _formatXLabel(label) {
+  /**
+   * Creates an unbound function that calls the given formatter on this {@link ChartJsRenderer}. The context of this
+   * unbound function is passed to the formatter as argument.
+   */
+  _bindLabelFormatter(formatter) {
```

Comment:
The method is called "bind (something)", but it returns something that is *not* bound. Isn't that a bit confusing? Unfortunately, I have no better ideas... Maybe `_prepareLabelFormatter` or `_createLabelFormatter`? But since it is internal, it probably doesn't matter.